<ui version="4.0" >
 <class>MediaflyPlay</class>
 <widget class="QWidget" name="MediaflyPlay" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>440</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Mediafly now playing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QStackedWidget" name="m_stackedWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="mf::PlayAudio" name="m_audio" />
       <widget class="mf::PlayVideo" name="m_video" />
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_statusLabel" >
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NTimeBar" name="m_progressBar" />
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="font" >
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>?</string>
         </property>
         <property name="margin" >
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_playStateLabel" >
         <property name="font" >
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>></string>
         </property>
         <property name="margin" >
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mf::PlayAudio</class>
   <extends>QWidget</extends>
   <header>PlayAudio.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mf::PlayVideo</class>
   <extends>QWidget</extends>
   <header>PlayVideo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NTimeBar</class>
   <extends>QProgressBar</extends>
   <header>ntimebar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
